defi/src/protocols/parentProtocols.ts

```diff
@@ -3432,6 +3432,7 @@ const parentProtocols: IParentProtocol[] = [
     chains: [],
     twitter: "BalancedDAO",
     github: ["balancednetwork"],
+    stablecoins: ["bnusd"],
```

Comment:
do we have an adapter for this on the stablecoins repo?

Reply:
Currently, the stablecoin is being displayed as a seperate project. 

- Adapter here: https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/balanced-bnUSD/index.js 

I want to follow [Maker example](https://defillama.com/protocol/maker) and simply have a Stablecoin Info tab under the parent project listing of [Balanced](https://defillama.com/protocol/balanced#information)